source/dub/packagemanager.d

```diff
@@ -39,9 +39,20 @@ class PackageManager {
 
 	this(NativePath user_path, NativePath system_path, bool refresh_packages = true)
 	{
-		m_repositories.length = LocalPackageType.max+1;
-		m_repositories[LocalPackageType.user] = Repository(user_path ~ "packages/");
-		m_repositories[LocalPackageType.system] = Repository(system_path ~ "packages/");
+		m_repositories = [
+			Repository(user_path ~ "packages/"),
+			Repository(system_path ~ "packages/")];
+
+		if (refresh_packages) refresh(true);
+	}
+
+    this(NativePath package_path, NativePath user_path, NativePath system_path, bool refresh_packages = true)
+	{
+		m_repositories = [
+			Repository(package_path~ ".dub/packages/"),
```

Comment:
buildPath

Reply:
It is working fine on windows, as placement is of type NativePath